installer/cli/microk8s.py

```diff
@@ -142,6 +142,12 @@ def install(args) -> None:
                 echo.error('Cannot continue without enabling Hyper-V')
                 exit(1)
 
+    if platform == 'darwin':
+        if not MacOS().is_enough_space():
```

Comment:
Given that the disk space is typically dynamically allocated do we really want to stop the installation? Is disk space pre-allocated with multipass on win and mac?

Even if we want to ensure the disk space can be pre-allocated, shouldn't we check for the disk space the user asked for with the `--disk` argument (that may be the default size of 50GB)?

Reply:
Have a look at this https://github.com/canonical/multipass/issues/62#issuecomment-360060429 . A warning would be more suitable.

Reply:
Yeah, we’ve been asked to do it this way and it’s already in the Windows code.